maquetado/setup/scss/tech.scss

```diff
@@ -8,21 +8,27 @@
   height: 300px;
   display: flex;
   flex-direction: column;  
-  align-items: center;
-
+  align-items: center;  
+  
   &::before {
     content: '';
     border: 2px solid  $green;
     position: absolute;
-    right: 5px;
-    bottom: 5px;
+    left: -7px;
```

Comment:
Why did you change it for negative values? Maybe you can use a positive `bottom` and `right`

Reply:
Because i needed to use the same attributes into the transition.